ansibullbot/triagers/ansible.py

```diff
@@ -2450,4 +2450,4 @@ def set_resume(self, repo, number):
         }
         resume_file = os.path.join(self.cachedir_base, u'resume.json')
         with open(resume_file, 'wb') as f:
-            f.write(json.dumps(data, indent=2))
+            f.write(json_dumps(data))
```

Comment:
same as above I guess

Reply:
Yep, all the f.write's should do to_bytes() on the data.